core/rpc/src/rpc_impl/transfer.rs

```diff
@@ -849,13 +929,82 @@ where
         let current_epoch = CURRENT_EPOCH.read().clone();
         (current_epoch - epoch) > self.cheque_since
     }
+
+    fn update_tx_view_change_cell(
+        &self,
+        tx_view: ckb_jsonrpc_types::TransactionView,
```

Comment:
Use `ckb_jsonrpc_types::TransactionView` at the begining of this file.

Reply:
updated.